Work orders manager enhancement (#3781)
DFHack/dfhack: docs/about/Removed.rst

  :local:
  :depth: 1

.. _workorder-recheck:

workorder-recheck
=================

> Tool to set 'Checking' status of the selected work order forcing manager to reevaluate its
> conditions. Merged into `orders`.

Reviewer: ```suggestion
Tool to set 'Checking' status of the selected work order, allowing conditions to be
reevaluated. Merged into `orders`.
```
